server/src/main/java/org/spine3/server/command/CommandBus.java

```diff
@@ -280,8 +280,13 @@ private void schedule(Command command) {
         this.messageValidator = messageValidator;
     }
 
-    private void store(Command request) {
-        commandStore.store(request);
+    /**
+     * Stores a command to the storage.
+     *
+     * @param command a command to store
+     */
+    public void store(Command command) {
```

Comment:
We must not expose this method. If we do, anybody who can post to the Command Bus would be able to store the command.